packages/data-models/appConfig.ts

```diff
@@ -119,9 +125,20 @@ const activeRoute = (location: Location) => {
   return path;
 };
 
-const APP_FOOTER_DEFAULTS = {
-  templateName: null as string | null,
-  background: "primary" as IHeaderFooterBackgroundOptions,
+interface IAppConfigFooterBase {
```

Comment:
nit(non-blocking): I'm not sure if we really need to keep two type definitions for base and extending with deprecated. I think you can probably just define a single config interface that includes the marked deprecations - I think probably a bit easier to mentally parse if everything is in one place